### PR TITLE
update Hang In There

### PR DESCRIPTION
**Pull request by sertmer**
I've given the spec a once-over. Not too many changes - Mostly just removing references to classes and replacing them with reference to a `createPoster` function. 

I also added some notes about functions purity. 

I haven't updated the starter repo yet - but it seems like it'll be a simple change from providing a `Poster` class to providing a `createPoster` function that will take in an imageURL, Title, and Quote. That'll come next! 

Let me know if it needs any changes!

### CODE REVIEW

**Comment by sertmer** [comment]
I've also updated the starter repo - you can view that PR here: https://github.com/turingschool-examples/hang-in-there-boilerplate/pull/298

File: projects/module-1/hang-in-there-v2.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Hang In There - Pair
 ## Learning Goals
 
 * Write clean, DRY JavaScript
-  * Use a provided class by creating object instances using the `new` keyword
+  * Use a provided createPoster function to create your `poster`'s
```
Comment (nicktu12) [inline projects/module-1/hang-in-there-v2.md]:
```suggestion
  * Use a provided createPoster function to create your `poster`s
```

File: projects/module-1/hang-in-there-v2.md [hunk 2]
```diff
@@ -185,16 +185,16 @@ This project gives the team room to practice the following JavaScript concepts:
...
 <section class="answer">
-### OOP competency examples
+### Functional Programing competency examples
 
-- Project makes use of the provided class, creating instances to encapsulate & store unique data
-- Team considers when and how to create new instances or update existing instances, coming to a clear and consistent decision
-- Team effectively creates/accesses instance properties when applicable
-- Team effectively invokes instance methods when applicable
+- Project makes use of the provided `createPoster` function, creating objects to encapsulate & store unique data
+- Team considers when and how to create poster objexts or update existing poster objects, coming to a clear and consistent decision
```
Comment (nicktu12) [inline projects/module-1/hang-in-there-v2.md]:
```suggestion
- Team considers when and how to create poster objects or update existing poster objects, coming to a clear and consistent decision
```

**Comment by nicktu12** [comment]
Nice work @sertmer ! Added just a few spelling fixes. Have you explored the github `add suggestion` button - really cool way to make a suggestion + set up a PR commit at the same time!

**Comment by sertmer** [comment]
> Nice work @sertmer ! Added just a few spelling fixes. Have you explored the github `add suggestion` button - really cool way to make a suggestion + set up a PR commit at the same time!

Woah! I didn't know about that, but it was super helpful here. Thanks for your eyes and time, Nick!

**Pull request merged**